src/static/js/utils/process-json.js

```diff
@@ -1,53 +1,5 @@
 'use strict';
 
```

Comment:
I had a polyfill previously, but what's weird is that I tested in in IE8 in sauce and it all rendered fine. (`npm test` couldn't connect to sauce to run those tests at the time.)

I'll look into it today! A polyfill is a fine fix IMO. :)